views/config_notifications.mako

```diff
@@ -1593,6 +1596,7 @@
                                             <span class="component-desc">
                                                 <input type="checkbox" name="trakt_sync_remove" id="trakt_sync_remove" ${'checked="checked"' if app.TRAKT_SYNC_REMOVE else ''}/>
                                                 <p>Remove an Episode from your Trakt Collection if it is not in your Medusa Library.</p>
+                                                <p><b>Note:</b> Don't enable this setting if you use Trakt addon in Kodi (or other script) that syncs library.</p>
```

Comment:
Don't enable this setting if you use the Trakt addon for Kodi or any other script that syncs your library.